ci-operator/step-registry/windows/e2e/operator/test/byoh/windows-e2e-operator-test-byoh-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# TODO: move jq installation to Dockerfile.ci
+mkdir -p "${SHARED_DIR}"/bin
+curl -sfL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 --output "${SHARED_DIR}"/bin/jq
+chmod +x "${SHARED_DIR}"/bin/jq
+export PATH=$PATH:"${SHARED_DIR}"/bin
```

Comment:
This should not be done here, it is not necessary to do the setup steps in this script. this needs to be done at the Dockerfile level if jq is actually necessary.

Reply:
@jrvaldes the last I checked I thought you were injecting jq into the image. Why did you move away from that approach? If you are not able to get it to work, please engage the test-platform team and figure it out. We are already downloading `oc` and we should be moving away from that instead of adding another bad example.

Reply:
I couldn't make the release image available in the registry. Hence, the initContainer [failed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/24911/rehearse-24911-pull-ci-openshift-windows-machine-config-operator-release-4.11-platform-none-vsphere-e2e-operator/1488603282610851840/artifacts/build-resources/events.json#:~:text=Back%2Doff%20pulling%20image)  while pulling the image for  `cli-jq`. Will follow up with  test-platform team.

Reply:
Moved `jq` installation to CI Dockerfile. See https://github.com/openshift/windows-machine-config-operator/pull/900.